Automattic/sensei
# Update build and package script methods

Sets it up for using woorelease.

### Changes proposed in this Pull Request

* Switches to using composer's archive command to build the release package.
* Moves over the gulp command to update select2 assets.
* Removes the gulpfile.js, gulp, and its dependencies.
* Updates documentation.

### Testing instructions

* Ensure ZIP package (`npm run build`) matches that of previous build (`gulp package-unsafe`).

`master`: [sensei-lms-old.zip](https://github.com/Automattic/sensei/files/5129078/sensei-lms-old.zip)
This PR: [sensei-lms-new.zip](https://github.com/Automattic/sensei/files/5129077/sensei-lms-new.zip)

`diff --brief --recursive --new-file sensei-lms sensei-lms-old` shows no difference.
